### ISSUE TITLE
convertPathsToSystemSpecific

### ISSUE THREAD

**Issue by maisk** [tomusdrw/grunt-sync#27]
```javascript
 var convertPathsToSystemSpecific = function(paths) {
    return paths.map(function(filePath) {
      return path.join.apply(path, filePath.split('/'));
    });
  };
```

convertPathsToSystemSpecific('/tmp/test')
returns tmp/test

no slash (/) at the start of the string
is this correct?

thanks

**Comment by tomusdrw** [comment]
Hi!
I believe it's not correct at all. I assumed that path.join would work like [].join and the problem is that
```javascript
require('path').join('', 'tmp', 'test') === 'tmp/test'
```

Can you think of any better method for checking if the path is relative / absolute?
My ideas:
```javascript
filePath[0] === '/'
grunt.file.isPathAbsolute
```

**Comment by maisk** [comment]
i want to clarify that we discuss  about
the method convertPathsToSystemSpecific of the grunt-sync project
is this correct?

in my occassion i want to syncronize the src to /something
and this method returns "something"  (mean without /)
as a result i get an error

**Comment by tomusdrw** [comment]
Fixed in 0.4.1
